src/openloco/windows/terraform.cpp

```diff
@@ -0,0 +1,49 @@
+#include "../interop/interop.hpp"
+#include "../ui/WindowManager.h"
+
+using namespace openloco::interop;
+
+namespace openloco::ui::windows::terraform
+{
+    window* open()
+    {
+        registers regs;
+        call(0x004BB4A3, regs);
+        return (window*)regs.esi;
+    }
+
+    // 0x004BB566
+    void open_clear_area()
+    {
+        auto terraform_window = open();
+        terraform_window->call_on_mouse_up(4);
+    }
+
+    // 0x004BB546
+    void open_adjust_land()
```

Comment:
You can implement the shortcuts for these as well

Reply:
Bit outside the scope of where I want to take this PR. Seems like something for #271 to tackle.